### PR TITLE
[Feat] : added Notes page UI 

### PR DESCRIPTION
**Pull request by anjali1102**
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
```
Comment (govindappaarun) [inline public/index.html:9]: suggestion: update this meta also

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (govindappaarun) [inline public/manifest.json:3]: suggestion: even this content too

**Review by govindappaarun — APPROVED** [review]

File: src/components/Card.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    },
+    {
+      primaryColor: "#F48687",
+      secondaryColor: "#FDF1F1",
+    },
+    {
+      primaryColor: "#B964F7",
+      secondaryColor: "#F3F0FD",
+    },
+  ];
```
Comment (jeetsdev) [inline src/components/Card.js:27]: Not able to understand what are you trying to do here...Can you just give a short description here pls?

File: src/components/Notes.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        <button
+          onClick={() => {
+            setModal(true);
+          }}
+        >
+          Create Note
+        </button>
+      </div>
+      <div className="task-container">
+        {taskList && taskList.map((obj,index) => <Card taskObj={obj} index={index} />)}
```
Comment (jeetsdev) [inline src/components/Notes.js:42]: Add the `key` attribute here.

**Review by jeetsdev — APPROVED** [review]
Just added some suggestions, else looks good to merge.

File: src/components/Card.js (hunk 3 above)
Comment (anjali1102) [reply src/components/Card.js:27]: for random color

**Pull request merged**